code/game/objects/items/weapons/storage/storage_base.dm

```diff
@@ -382,6 +382,12 @@
 					to_chat(usr, "<span class='warning'>[src] rejects [I] because of its contents.</span>")
 				return FALSE
 
+	if(istype(src, /obj/item/storage/belt) && isstorage(loc)) //You shouldnt be able to bypass belt in bag storage
+		if(!(istype(src, /obj/item/storage/belt/military/traitor) || istype(src, /obj/item/storage/belt/utility/chief)))
```

Comment:
Why not check for `storable` here instead of making specific `istype` checks?

Reply:
done